src/main/java/rs/teslaris/core/harvester/event/Event.java

```diff
@@ -0,0 +1,54 @@
+package rs.teslaris.core.harvester.event;
+
+import java.util.Date;
+import java.util.List;
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlAttribute;
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlRootElement;
+import javax.xml.bind.annotation.XmlType;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+import rs.teslaris.core.harvester.common.MultilingualContent;
+
+@XmlType(name = "TEvent", namespace = "https://www.openaire.eu/cerif-profile/1.1/")
+@XmlAccessorType(XmlAccessType.FIELD)
+@XmlRootElement(name = "Event")
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@ToString
+public class Event {
+
+    @XmlAttribute(name = "id")
+    private String id;
+
+    @XmlElement(name = "Type")
+    private EventType eventType;
+
+    @XmlElement(name = "Name")
+    private MultilingualContent multilingualContent;
```

Comment:
please check whether this should be a list.  I can't remember how we defined this in the model